src/common.h

```diff
@@ -129,11 +129,12 @@ int ishexlcn(const char *q, int n);
 size_t hexlen(const char *q, int *extra_chars);
 size_t hexlenl(const char *q, int *extra_chars); /* lower cased only */
 size_t hexlenu(const char *q, int *extra_chars); /* upper cased only */
-/* is this a valid string for atoi() ONLY positive numbers are valid */
+/*Is this a valid number and within the range of [0...999999999].
+ *ONLY positive numbers are valid. */
 int isdec(const char *q);
-/* is this a valid string for atoi() */
+/*Is this a valid number and within the range of [0...999999999]*/
 int isdec_negok(const char *q);
-/* is this a valid string for atou()?  atou() func == sprintf("%x",&val) */
+/* Is this a valid number and within the range of [0...999999999]*/
```

Comment:
Comments no longer reflect function

Reply:
@magnumripper oops, sorry I forgot that. And In my isdec_negok function, I forgot something too. I just remembered. Good morning everyone. 🤦